### PR TITLE
radar(ci-cd): moves GHA to adopt and other CI-CD to hold

### PR DESCRIPTION
**Pull request by Gio-Jimdo**
Travis CI an Circle CI should no longer be used. All existing uses should migrate to GitHub Actions.

### CODE REVIEW

**Comment by pezhman32** [comment]
@Gio-Jimdo while viewing the file, github complains:
```
We can make this file beautiful and searchable if this error is corrected: It looks like row 5 should actually have 6 columns, instead of 5. in line 4.
```

**Comment by oerd** [comment]
> @Gio-Jimdo while viewing the file, github complains:
> 
> ```
> We can make this file beautiful and searchable if this error is corrected: It looks like row 5 should actually have 6 columns, instead of 5. in line 4.
> ```

@Gio-Jimdo added the `url` field, all existing fields require a trailing `,` to match the field number

**Comment by bijancn** [comment]
For future reference: this is the corresponding adoption task https://jimplan.atlassian.net/wiki/x/DgB-sg

File: data/backend.csv [hunk 1]
```diff
@@ -1,4 +1,7 @@
-name,ring,quadrant,isNew,description
+name,ring,quadrant,isNew,description,url
```
Comment (cmuelhau) [inline data/backend.csv:1]: the addition of `url` seems to break the nice table view in github. Most likely, you'd have to add an empty column for all other entries

**Review by cmuelhau — CHANGES_REQUESTED** [review]

File: data/backend.csv (hunk 1 above)
Comment (Gio-Jimdo) [reply data/backend.csv:1]: Yup, I fell into the very trap I mentioned of ensuring the CSV is rendering before you PR. This convo has validated my expectation that we'd catch even formatting issues thru this process. I'll fix that now.

**Review by cmuelhau — APPROVED** [review]

**Pull request merged**